### PR TITLE
Add Lifeline notebook

### PR DESCRIPTION
**Pull request by MaxGhenis**
* Lifeline notebook
* Lifeline parameter metadata for PolicyEngine
* Hooks up existing income concepts to Lifeline income variable (`irs_gross_income`)
* Adds CalifornIa CVRP to list of benefits so it flows to net income
* Other minor changes

### CODE REVIEW

**Comment by MaxGhenis** [comment]
Ready for review. BTW I included this chart in the notebook to show how categorical eligibility affects Lifeline.
image

File: openfisca_us/variables/irs/irs_gross_income.py [hunk 1]
```diff
@@ -9,3 +9,12 @@ class irs_gross_income(Variable):
     documentation = "Gross income as defined in the Internal Revenue Code"
     definition_period = YEAR
     reference = "https://www.law.cornell.edu/uscode/text/26/61"
+
+    def formula(person, period, parameters):
+        COMPONENTS = [
```
Comment (nikhilwoodruff) [inline openfisca_us/variables/irs/irs_gross_income.py:14]: Minor but could be a parameter?

File: openfisca_us/variables/irs/irs_gross_income.py [hunk 2]
```diff
@@ -9,3 +9,12 @@ class irs_gross_income(Variable):
...
+            "employment_income",
+            "self_employment_income",
+            "dividend_income",
+            "interest_income",
+        ]
+        return add(person, period, COMPONENTS)
```
Comment (nikhilwoodruff) [inline openfisca_us/variables/irs/irs_gross_income.py:20]: Also filed https://github.com/PolicyEngine/openfisca-tools/issues/23

**Review by nikhilwoodruff — APPROVED** [review]

File: openfisca_us/variables/irs/irs_gross_income.py (hunk 1 above)
Comment (MaxGhenis) [reply openfisca_us/variables/irs/irs_gross_income.py:14]: Yeah or actually should I just use `c00100`?

**Pull request merged**